src/background/services/openPayments.ts

```diff
@@ -906,7 +908,16 @@ export class OpenPaymentsService {
 
   async reconnectWallet() {
     try {
+      const { walletAddress } = await this.storage.get(['walletAddress']);
+      const tabId = await this.addPublicKeyToWallet(walletAddress!);
+      this.setConnectState('connecting');
       await this.rotateToken();
+      await this.redirectToWelcomeScreen(
+        tabId!,
```

Comment:
The tab should be open when we redirect to the welcome screen, otherwise the redirect will throw.

CC @sidvishnoi: should we still perform the `tabId` check as well? If the user closes the tab we can open a new one if needed.